runserver.sh

```diff
@@ -20,4 +20,4 @@ python3 manage.py create_test_users
 python3 manage.py create_test_actions
 
 echo "-------- Run Server"
-python3 manage.py runserver
+python3 manage.py runserver 0.0.0.0:8000
```

Comment:
Does it matter?

Reply:
The default IP address, 127.0.0.1, is not accessible from other machines on network. To make the development server viewable to other machines on the network, need to use its own IP address (e.g. 192.168.2.1) or 0.0.0.0.

Reply:
Out of curiosity, which other machines will need to access your Dev server?